map_server/src/image_loader.cpp

```diff
@@ -43,7 +43,7 @@
 #include <SDL/SDL_image.h>
 
 #include "map_server/image_loader.h"
-#include <tf/tf.h>
+#include <tf2/LinearMath/Quaternion.h>
```

Comment:
My recommendation would be to add a dependency on rosdep `bullet` and use the btQuaternion directly. It's a strictly smaller dependency than `tf`. You should be able to substitute `btQuaternion` for `tf::Quaternion` and change the includet to be `<btQuaternion.h>` and use [`setEulerZYX`](http://bulletphysics.org/Bullet/BulletFull/classbtQuaternion.html#a808b430136b4a8d1e7120134af14a1ff) (note the arguments are x, y, z in the RPY case and need to be reversed.)